### PR TITLE
Added animations for inner layer

### PR DESCRIPTION
**Pull request by JackieW001**
## Overview

Inner layer is animated.

## Changes Made

Added sprite sheet for inner layer and converted inner layer Texture nodes to Animation nodes.

## Next Steps (delete if not applicable)

Include animations for outer layer and core.

### CODE REVIEW

File: source/CIPlanetModel.h [hunk 1]
```diff
@@ -142,18 +145,14 @@ class PlanetModel {
...
-    /**
-     * Returns the scene graph node for this planet
-     *
-     * @return the image texture for this planet
-     */
+   
     const std::shared_ptr<cugl::scene2::SceneNode> getPlanetNode() const {
         return _planetNode;
     }
+
```
Comment (steiner26) [inline source/CIPlanetModel.h:155]: Nit: can we avoid double new lines?

File: assets/json/assets.json [hunk 2]
```diff
@@ -41,6 +41,20 @@
...
+                    "span": 91,
+                    "cols": 10,
+                    "frame": 0,
+                    "anchor": [0.5,0.5]
+                  },
+                  "layout": {
+                    "x_anchor" : "center",
+                    "y_anchor" : "middle"
+                  }
+                },
```
Comment (steiner26) [inline assets/json/assets.json:57]: Is there a reason why this needed to be added to the json file if we're just going to programmatically make new layer nodes in the code?

File: source/CIPlanetModel.cpp [hunk 3]
```diff
@@ -30,16 +30,19 @@
  * @param unlocked The texture on the outside of a locked ring
  */
 void PlanetModel::setTextures(const std::shared_ptr<cugl::Texture>& core,
-                 const std::shared_ptr<cugl::Texture>& ring,
-                 const std::shared_ptr<cugl::Texture>& unlocked,
-                 const std::shared_ptr<cugl::Texture>& locked) {
+                              const std::shared_ptr<cugl::scene2::AnimationNode>& ring,
```
Comment (steiner26) [inline source/CIPlanetModel.cpp:33]: Same comment here, why does did this need to be changed to an AnimationNode? Couldn't we just pass in the texture sheet?

File: source/CIPlanetNode.h [hunk 4]
```diff
@@ -84,4 +98,5 @@ class PlanetNode : public cugl::scene2::PolygonNode {
     }
 };
 
+
```
Comment (steiner26) [inline source/CIPlanetNode.h:101]: Same here, can we avoid double newlines?

**Review by steiner26 — APPROVED** [review]
Looks good to me, I was just wondering why we needed to get the ring texture as an animation node rather than just getting the texture. If we're going to make new animation nodes in the code (and not the json) it seems a bit redundant to add the innerRing to the scene graph

**Review by wlong123 — APPROVED** [review]
🚀

**Pull request merged**